pathways/transcribe.js

```diff
@@ -4,7 +4,7 @@ export default {
     inputParameters: {
         file: ``,
```

Comment:
It would be nice if Transcribe pathway took the optional Whisper argument to translate to English in one step since it's apparently good at that.

Reply:
added optional language parameter